conf/config-schema.json

```diff
@@ -0,0 +1,16 @@
+{
+    "type": "object",
+    "properties": {
+        "root": { "type": "boolean" },
+        "globals": { "type": ["object", "array"] },
+        "global": { "type": ["object", "array"] },
```

Comment:
`globals` vs. `global` were done before version 1.0.0 because JSHint supported both. So it was done for backwards compatibility. I would be 👎 on removing either one at this point. This would be a major breaking change from my perspective. I'm not sure why `global` is not used in config, it should be, it might've been lost during refactoring. But inline comments accept both `globals` and `global`